models/src_linkedin.yml

```diff
@@ -308,5 +308,78 @@ sources:
           - name: external_website_conversions
             description: The actions taken on your website that you've defined as valuable to your business after clicking on LinkedIn ads.
 
+          - name: one_click_leads
+            description: Leads submitted after clicking on LinkedIn ads.
+
+      - name: geo
+        description: Contains geographic data for LinkedIn Ads, including IDs and names of locations.
+        identifier: "{{ var('linkedin_ads_geo_identifier', 'geo') }}"
+        config:
+          enabled: "{{ var('ad_reporting__linkedin_ads_enabled', true) and var('linkedin_ads__using_geo', true) }}"
+        columns:
+          - name: id
+            description: Unique identifier for the geographic entity, used for joins in reporting tables.
+          - name: value
+            description: Name of the geographic entity.
+
+      - name: monthly_ad_analytics_by_member_country
+        description: Monthly LinkedIn Ads performance metrics aggregated by member country and campaign.
+        identifier: "{{ var('linkedin_ads_monthly_ad_analytics_by_member_country_identifier', 'monthly_ad_analytics_by_member_country') }}"
+        config:
+          enabled: "{{ var('ad_reporting__linkedin_ads_enabled', true) and var('linkedin_ads__using_monthly_ad_analytics_by_member_country', true) }}"
+        columns:
+          - name: campaign_id
+            description: Identifier of the LinkedIn Ads campaign.
+          - name: month
+            description: Month of the reporting data, aliased as `date_month`.
+          - name: clicks
+            description: The count of chargeable clicks.
+          - name: member_country
+            description: ID representing the country of the LinkedIn members who interacted with the ad.
+          - name: impressions
+            description: This is the count of "impressions" for Direct Ads and Sponsored Updates and "sends" for InMails.
+          - name: conversion_value_in_local_currency
+            description: The value generated by your conversions, displayed in your LinkedIn account's local currency.
+          - name: cost_in_local_currency
+            description: The cost of the ads in the local currency.
+          - name: cost_in_usd
+            description: The cost of the ads in the USD.
+          - name: external_website_conversions
+            description: Number of conversions that occurred on an external website as a result of the ad.
+          - name: one_click_leads
+            description: Number of one-click leads generated from the ad campaign.
+          - name: external_website_conversions
+            description: The actions taken on your website that you've defined as valuable to your business after clicking on LinkedIn ads.
+          - name: one_click_leads
+            description: Leads submitted after clicking on LinkedIn ads.
```

Comment:
These columns are duplicated. I think the second set of columns have better descriptions, so I'd opt to keep those, but will leave up to you!

Reply:
Unsure how this happened. Removed.